src/analysis/complex/phragmen_lindelof.lean

```diff
@@ -140,13 +140,13 @@ begin
     ⟨(a + b) / 2, (b - a) / 2, by ring, by ring⟩,
   have hab : a - b < a + b, from hza.trans hzb,
   have hb : 0 < b,
-    by simpa only [sub_eq_add_neg, add_lt_add_iff_left, neg_lt_self_iff] using hab,
+  { simpa only [sub_eq_add_neg, add_lt_add_iff_left, neg_lt_self_iff] using hab },
```

Comment:
The indented `by` is accepted mathlib style I believe, so I wouldn't change this. (At least an indented `from` is.)